packages/stateful/proposal-module-adapter/adapters/CwdProposalMultiple/hooks/useProposalExecutionTxHash.ts

```diff
@@ -0,0 +1,28 @@
+import { constSelector, useRecoilValue } from 'recoil'
+
+import { proposalExecutionTXHashSelector } from '@dao-dao/state'
+import { Status } from '@dao-dao/types/contracts/CwdProposalSingle.common'
```

Comment:
probably worth moving this to `types/contracts/common.ts` and renaming to a general `ProposalStatus`